cmd/massa-wallet/thyra-plugin-wallet.go

```diff
@@ -44,11 +35,35 @@ func StartServer(app *fyne.App, port int) {
 	server := restapi.NewServer(massaWalletAPI)
 	server.ConfigureAPI()
 
-	// Set the port to listen on to the passed-in port
-	server.Port = port
+	ln, err := server.HTTPListener()
+	if err != nil {
+		log.Fatalln(err)
+	}
+	registerPlugin(ln)
 
 	if err := server.Serve(); err != nil {
-
 		log.Fatalln(err)
 	}
 }
+
+func registerPlugin(ln net.Listener) {
```

Comment:
This code is the very same for all module.
It should be define at the hello-world level and a function should be imported here to avoid copy/paste

Reply:
you right, I have twisted hello world a bit here https://github.com/massalabs/thyra-plugin-hello-world/pull/8